### PR TITLE
keystone custom message proto

### PR DESCRIPTION
**Pull request by patrickhuie19**
Supports 

https://github.com/smartcontractkit/chainlink/pull/14510

### CODE REVIEW

File: pkg/beholder/pb/keystone/keystone_custom.proto [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/beholder/pb/keystone";
+
+package pb;
+
+message KeystoneCustomMessage {
+  string msg=1;
+  // https://protobuf.dev/programming-guides/proto3/#maps
+  // In go: if Value is empty for a key, nothing will be serialized
+  map<string, Value> labels=2;
```
Comment (cedric-cordenier) [inline pkg/beholder/pb/keystone/keystone_custom.proto:11]: @patrickhuie19 Can we import the values library here instead? We shouldn't need to duplicate this type

File: pkg/beholder/pb/keystone/keystone_custom.proto [hunk 2]
```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/beholder/pb/keystone";
+
+package pb;
+
+message KeystoneCustomMessage {
```
Comment (cedric-cordenier) [inline pkg/beholder/pb/keystone/keystone_custom.proto:7]: For the required Keystone labels (workflowOwner, id, etc), are these expected to be added to the labels map?

File: pkg/beholder/pb/keystone/keystone_custom.proto [hunk 3]
```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```
Comment (cedric-cordenier) [inline pkg/beholder/pb/keystone/keystone_custom.proto:1]: Reading the thread you linked me to yesterday, does it make sense to put this here? As opposed to in pkg/capabilities/pb?

File: pkg/beholder/pb/keystone/keystone_custom.proto (hunk 2 above)
Comment (patrickhuie19) [reply pkg/beholder/pb/keystone/keystone_custom.proto:7]: Yes, thats the expectation. I prefer the flexibility and scalability of this approach (a generic labels map) over an explicit definition of each field like I had before.

File: pkg/beholder/pb/keystone/keystone_custom.proto (hunk 3 above)
Comment (patrickhuie19) [reply pkg/beholder/pb/keystone/keystone_custom.proto:1]: Is capabilities the right dir for it? A msg with some labels could be used pretty much everywhere. Mb that's more appropriate in a beholder general dir? Then other consumers can perhaps consume it and add custom fields

WDYT (@cll-gg)?

File: pkg/values/pb/values.pb.go [hunk 4]
```diff
@@ -494,7 +494,7 @@ func file_values_pb_values_proto_rawDescGZIP() []byte {
 }
 
 var file_values_pb_values_proto_msgTypes = make([]protoimpl.MessageInfo, 6)
-var file_values_pb_values_proto_goTypes = []interface{}{
+var file_values_pb_values_proto_goTypes = []any{
```
Comment (patrickhuie19) [inline pkg/values/pb/values.pb.go:497]: the `interface` to `any` updates unintentional, but afaik more idiomatic.

File: pkg/beholder/pb/keystone/keystone_custom.proto (hunk 1 above)
Comment (patrickhuie19) [reply pkg/beholder/pb/keystone/keystone_custom.proto]: 345595f38e0ec17b8a5f1f488aed12aca0414d86

File: pkg/beholder/pb/keystone/keystone_custom.proto (hunk 3 above)
Comment (patrickhuie19) [reply pkg/beholder/pb/keystone/keystone_custom.proto]: refactored to sit in pkg/beholder/pb

345595f38e0ec17b8a5f1f488aed12aca0414d86

**Review by krehermann — APPROVED** [review]

**Review by pkcll — APPROVED** [review]

**Review by MStreet3 — APPROVED** [review]

**Pull request merged**